maestrowf/maestro.py

```diff
@@ -61,17 +62,43 @@
 
 def status_study(args):
     """Check and print the status of an executing study."""
-    status = Conductor.get_status(args.directory)
-
-    if status:
-        print(tabulate.tabulate(status, headers="keys"))
-        return 0
+    #args_dir is the path specified
+    args_dir = str(args.directory)
+    args_dir = args_dir.replace("[", "")
```

Comment:
You don't need to check for these characters. It turns out that the shell is resolving the glob that's passed into Maestro command line before the code sees it. That's the reason why the `nargs="+"` fix works, it's because it's already a list of paths when Maestro sees it. You shouldn't be looking to remove the regex/glob patterns because you'll never actually see them.